MailManager/src/main/java/inc/softserve/EmailServiceModule.java

```diff
@@ -2,30 +2,30 @@
 
 import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
+import com.google.inject.matcher.Matchers;
 import com.google.inject.name.Named;
-import inc.softserve.common.EnvelopeTools;
-import inc.softserve.common.EnvelopeToolsImpl;
+import inc.softserve.annotations.*;
+import inc.softserve.common.*;

```

Comment:
Usually we try to avoid wildcard imports. I'd suggest to configure your IDE to not substitute explicit imports with wildcards. Do you mind changing it?